sdk/feature-management-applicationinsights-node/rollup.config.mjs

```diff
@@ -0,0 +1,47 @@
+// rollup.config.js
+import typescript from "@rollup/plugin-typescript";
+import dts from "rollup-plugin-dts";
+
+export default [
+  {
+    input: "src/index.ts",
+    output: [
+      {
+        dir: "dist/commonjs/",
+        format: "cjs",
+        sourcemap: true,
+        preserveModules: true,
+      },
+      {
+        dir: "dist/esm/",
+        format: "esm",
+        sourcemap: true,
+        preserveModules: true,
+      }
+    ],
+    plugins: [
+      typescript({
+        compilerOptions: {
+          "lib": [
+            "DOM",
```

Comment:
Do we need DOM in this node package?

Reply:
I reused the rollup.config left by @Eskibear . I guess this is from some template? Will figure it out later. But in this PR, I don't mind keeping them.